src/Editor.tsx

```diff
@@ -49,7 +58,34 @@ export class Editor extends React.Component<{
               <path d="M17 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V7l-4-4zm-5 16c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3zm3-10H5V5h10v4z" />
             </svg>
           </button>
-          <button aria-label="Load">
+          <input
+            type="file"
+            id="fileInput"
+            accept=".lisp"
+            style={{ display: "none" }}
+          />
+          <button
+            onClick={() => {
+              const input = document.getElementById("fileInput");
```

Comment:
I'm still not sure why we need both `input` and `inputFile` (below). Aren't they the same?

Reply:
You're right, they are. I was wrong in thinking the query selector was necessary as well.